### PR TITLE
docs: Remove extra symbols

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

**Comment by caicancai** [comment]
cc @chaokunyang

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -132,12 +132,12 @@ See the [Debugging C++](docs/cpp_debug.md) doc.
 Enable core dump on Macos Monterey 12.1:
 
 ```bash
-/usr/libexec/PlistBuddy -c "Add :com.apple.security.get-task-allow bool true" tmp.entitlements     
+/usr/libexec/PlistBuddy -c "Add :com.apple.security.get-task-allow bool true" tmp.entitlements
 codesign -s - -f --entitlements tmp.entitlements /Users/chaokunyang/anaconda3/envs/py3.8/bin/python
 ulimit -c unlimited
 ```
 
-... then, run the code:
```
Comment (chaokunyang) [inline CONTRIBUTING.md:140]: then run the code:

**Comment by caicancai** [comment]
thank your review

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**